modelon/impact/client/entities/project.py

```diff
@@ -0,0 +1,245 @@
+    ):
+        self._project_id = project_id
+        self._project_definition = project_definition
+        self._project_sal = project_service
+
+    def __repr__(self):
+        return f"Project with id '{self._project_id}'"
+
+    def __eq__(self, obj):
+        return (
+            isinstance(obj, Project)
+            and obj._project_id == self._project_id
+            and obj._project_definition == self._project_definition
+        )
+
+    @property
+    def id(self) -> str:
+        """Project id"""
+        return self._project_id
+
+    def delete(self):
+        """Deletes a project.
+
+        Example::
+
+            project.delete()
+        """
+        self._project_sal.project_delete(self._project_id)
+
+    def _get_project_content(self, content):
+        return ProjectContent(content, self._project_id, self._project_sal)
+
+    def _refresh(self):
+        self._project_definition = ProjectDefinition.from_dict(
+            self._project_sal.project_get(self._project_id)['definition']
+        )
+
+    def get_contents(
+        self, content_type_filter: Optional[ContentType] = None
+    ) -> List[ProjectContent]:
+        """Get project contents.
+
+        Example::
+
+            project.get_contents(ContentType.MODELICA)
+        """
+        self._refresh()
+        contents = self._project_definition.content
+        if content_type_filter:
+            return [
+                self._get_project_content(content)
+                for content in contents
+                if content_type_filter == content.content_type
+            ]
+        return [self._get_project_content(content) for content in contents]
+
+    def upload_content(
+        self, path_to_content: str, content_type: ContentType
+    ) -> ProjectContent:
+        """Upload content to a project.
+
+        Parameters:
+
+            path_to_content --
+                The path for the content to be imported.
+
+            content_type --
+                The type of the imported content.
+
+        Example::
+            from modelon.impact.client import ContentType
+
+            project.upload_content('/home/test.mo', ContentType.MODELICA)
+        """
+        resp = self._project_sal.project_content_upload(
+            path_to_content, self._project_id, content_type.value
+        )
+        content = Content.from_dict(resp)
+        return self._get_project_content(content)
+
+    def upload_modelica_library(self, path_to_lib: str):
+        """Uploads/adds a non-encrypted modelica library or a modelica model to the project.
+
+        Parameters:
+
+            path_to_lib --
+                The path for the library to be imported. Only '.mo' or '.zip' file
+                extension are supported for upload.
+
+        Example::
+
+            project.upload_model_library('C:/A.mo')
+            project.upload_model_library('C:/B.zip')
+        """
+        if Path(path_to_lib).suffix.lower() == '.mol':
```

Comment:
Should this not be `not in ['.mo', '.zip']` rather than `== '.mol'` so the check is what the error bellow says.